pkg/client/devops/jclient/jenkins.go

```diff
@@ -0,0 +1,49 @@
+package jclient
+
+import (
+	"io/ioutil"
+
+	client "github.com/jenkins-zh/jenkins-client/pkg/core"
+	"gopkg.in/yaml.v2"
+)
+
+type JenkinsClient struct {
+	
+}
+type Config struct {
+	Jenkins JenkinsConfig `yaml:"devops,omitempty"`
+}
+type JenkinsConfig struct {
```

Comment:
At the same time, please check out existing JenkinsConfig: https://github.com/kubesphere/ks-devops/blob/0fe858fc6cf30523a18c456e7136a31459c70d9d/pkg/config/config.go#L81-L90